projects/1.20/assets/ex-deorum/exdeorum/lang/zh_cn.json

```
@@ -0,0 +1,176 @@
+{
+    "advancements.exdeorum.core.barrel.description": "使用木桶将有机物堆肥成泥土。",
+    "advancements.exdeorum.core.barrel.title": "那些只是绿色废品。",
+    "advancements.exdeorum.core.crook.description": "合成一个钩子，用它从树叶上收获更多的树苗。",
+    "advancements.exdeorum.core.crook.title": "给他一个钩子",
+    "advancements.exdeorum.core.root.description": "你在虚空中诞生——",
+    "advancements.exdeorum.core.root.title": "别往下看……",
+    "advancements.exdeorum.core.silk_worm.description": "你获得了一个蚕，然后用它感染一棵树的树叶，之后等待它滋生以获得线。",
```

```suggestion
    "advancements.exdeorum.core.silk_worm.description": "获得一只蚕，并用它感染一整颗树的树叶以获得线。",
```

```suggestion
    "advancements.exdeorum.core.silk_worm.description": "获得一只蚕，并用它感染一棵树以获得线。",
```


`一个蚕`不觉得别扭吗
原文没给那么多信息，最好不要加戏
我知道你想给不熟悉exnihilo的玩家做引导，但是咱还是尽量贴着原文走